components/Navbar.tsx

```diff
@@ -13,7 +13,7 @@ const Navbar: FC = () => {
     const NAVBAR_ELEMENTS: INavbarElement[] = [
         {
```

Comment:
```suggestion
        const NAVBAR_ELEMENTS: INavbarElement[] = [
        {
            title: 'Dashboard',
            url: '/',
        },
        {
            title: 'Candy Machines',
            url: '/candy-machines',
        },
    ]
```

Reply:
I think this is going to be modified in this [task](https://github.com/boxfish-studio/sugar-rush/issues/179)
Do i still change it?

Reply:
ups, I didn't see it, so it might be better to do it separately :pray: